package.json

```diff
@@ -5,47 +5,55 @@
   "author": "Greenpeace International",
   "license": "GPL-3.0",
   "scripts": {
-    "start": "wp-scripts start --config webpack.config.js",
-    "build": "wp-scripts build --config webpack.config.js --mode production",
+    "start": "wp-scripts start --config webpack.config.js --mode=development",
+    "build": "wp-scripts build --config webpack.config.js --mode=production",
     "lint:css": "wp-scripts lint-style"
   },
   "devDependencies": {
     "@commitlint/cli": "^17.7.2",
     "@greenpeace/dashdash": "^1.2.1",
-    "@playwright/test": "^1.37.1",
-    "@wordpress/components": "^25.7.0",
-    "@wordpress/e2e-test-utils-playwright": "^0.8.0",
-    "@wordpress/eslint-plugin": "^2.3.0",
-    "@wordpress/scripts": "3.3.0",
+    "@playwright/test": "1.37.1",
```

Comment:
Making sure we use this specific version of Playwright removes the dependency conflicts, we should be able to upgrade it again once we start using Node 18